Assets/HoloToolkit/Utilities/Scripts/SceneLauncher.cs

```diff
@@ -39,52 +60,40 @@ private void Start()
                 return;
             }
 
-            if (ReturnToSceneLauncherPrefab != null)
-            {
-                KeywordManager returnToSceneLauncher = Instantiate(ReturnToSceneLauncherPrefab);
-                DontDestroyOnLoad(returnToSceneLauncher);
-                if (returnToSceneLauncher.KeywordsAndResponses.Length > 0)
-                {
-                    // Set the response action of the first keyword to reload this scene.
-                    int sceneLauncherBuildIndex = SceneManager.GetActiveScene().buildIndex;
-                    UnityAction keywordAction = delegate
-                    {
-                        Debug.LogFormat("SceneLauncher: Returning to SceneLauncher scene {0}.", sceneLauncherBuildIndex);
-                        SceneManager.LoadScene(sceneLauncherBuildIndex, LoadSceneMode.Single);
-                        GameObject.Destroy(returnToSceneLauncher.gameObject);
-                    };
-                    returnToSceneLauncher.KeywordsAndResponses[0].Response.AddListener(keywordAction);
-                }
-            }
+            SceneLauncherBuildIndex = SceneManager.GetActiveScene().buildIndex;
 
             // Determine the size of the buttons. Instantiate one of them so that we can check its bounds.
-            SceneLauncherButton sceneButtonForSize = Instantiate(SceneButtonPrefab);
-            Collider sceneButtonForSizeCollider = sceneButtonForSize.GetComponent<Collider>();
+            var sceneButtonForSize = Instantiate(SceneButtonPrefab);
```

Comment:
I don't think this var is obvious, since I'd think it's a GameObject based on the Instantiate.